protocols/Leafcutter_intron_clustering.sh

```diff
@@ -0,0 +1,56 @@
+#MOLGENIS nodes=1 ppn=1 mem=34gb walltime=05:00:00
+
+#Parameter mapping
+#string intermediateDir
+#string project
+#string logsDir
+#string projectJobsDir
+#string strandedness
+#string sampleMergedBamExt
+#string leafcutterVersion
+#string python2Version
+
+#Load module
+module load "${leafcutterVersion}"
+module load "${python2Version}"
+module list
+
+# detect strand for RegTools
+STRANDED="$(num1="$(tail -n 2 "${strandedness}" | awk '{print $7'} | head -n 1)"; num2="$(tail -n 2 "${strandedness}" | awk '{print $7'} | tail -n 1)"; if (( $(echo "$num1 > 0.6" | bc -l) )); then echo "1"; fi; if (( $(echo "$num2 > 0.6" | bc -l) )); then echo "2"; fi; if (( $(echo "$num1 < 0.6 && $num2 < 0.6" | bc -l) )); then echo "0"; fi)"
+
+#detect number of conditions
+col=$(col="condition"; head -n1 "${projectJobsDir}/${project}.csv" | tr "," "\n" | grep -n $col)
+colArray=(${col//:/ })
+conditionCount=$(tail -n +2 "${projectJobsDir}/${project}.csv" | cut -d "," -f "${colArray[0]}" | sort | uniq | wc -l)
+
+echo -e "\nWith strandedness type: ${STRANDED},
+where (0 = unstranded, 1 = first-strand/RF, 2, = second-strand/FR)."
+
+rm -f "${intermediateDir}${project}_juncfiles.txt"
+cd "${intermediateDir}"
+for bamfile in $(ls *.${sampleMergedBamExt}); do
```

Comment:
It will give a shellcheck warning ;)
do this instead:
for bamfile in *".${sampleMergedBamExt}"